src/sentry/api/endpoints/organization_profiling_profiles.py

```diff
@@ -60,10 +62,10 @@ def get(self, request: Request, organization: Organization) -> HttpResponse:
 @region_silo_endpoint
 class OrganizationProfilingFlamegraphEndpoint(OrganizationProfilingBaseEndpoint):
     def get(self, request: Request, organization: Organization) -> HttpResponse:
-        if not features.has("organizations:profiling", organization, actor=request.user):
+        if not features.has("organizations:continuous-profiling", organization, actor=request.user):
```

Comment:
Was this change in feature flags intentional? This endpoint is for the transaction based profiles right?